### ISSUE TITLE
Inconsistency between installation instuctions and master config sample

### ISSUE THREAD

**Issue by liberforce** [buildbot/buildbot#3600]
Here are the installation instructions in the official documentation for 0.9.10:
http://docs.buildbot.net/0.9.10/tutorial/firstrun.html

The documentation only states to install this on the master:
`pip install 'buildbot[bundle]'
`
However in `master.cfg.sample` file we have:
```
# minimalistic config to activate new web UI
c['www'] = dict(port=8010,
                plugins=dict(waterfall_view={}, console_view={}, grid_view={}))
```
This will break because the grid view is tried to be used, but the corresponding plugin was not installed. 

So either the grid view is facultative and needs to be removed from the sample, or it is mandatory and the install instructions should be updated to install `buildbot-grid-view`, or `buildbot-grid-view` should be added to or as a dependency of `buildbot[bundle]`.

**Comment by Frodox** [comment]
It is fixed in upsteam with #3494 (bb-0.9.11 I think)

**Comment by liberforce** [comment]
Thanks, closing as a duplicate of #3494 then.
